tests/integration_tests/test_matchmaker.py

```diff
@@ -32,27 +33,66 @@ async def test_game_launch_message(lobby_server):
     await open_fa(proto1)
     msg2 = await read_until_command(proto2, "game_launch")
 
-    assert msg1["uid"] == msg2["uid"]
-    assert msg1["mod"] == msg2["mod"] == "ladder1v1"
-    assert msg1["mapname"] == msg2["mapname"]
-    assert msg1["team"] == 2
+    assert msg2["uid"] == msg1["uid"]
+    assert msg2["mod"] == msg1["mod"]
+    assert msg2["mapname"] == msg1["mapname"]
     assert msg2["team"] == 3
```

Comment:
Why are they team 2 and 3 instead of 1 and 2?

Reply:
Team 1 is the ffa team (shown as "-" in the lobby)